ps_customtext.php

```diff
@@ -322,11 +322,13 @@ public function renderWidget($hookName = null, array $configuration = [])
      */
     public function getWidgetVariables($hookName = null, array $configuration = [])
     {
-        $customText = new CustomText(1, (int) $this->context->language->id, (int) $this->context->shop->id);
+        $id_shop = (int) $this->context->shop->id;
+        $id_info = CustomText::getCustomTextIdByShop($id_shop);
```

Comment:
maybe you should test that `$id_info` is not false since the `getCustomTextIdByShop` method can return false

Reply:
Thanks. You are right. I fixed it.